src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -26,7 +42,27 @@ public class Pinecone {
         this.manageIndexesApi = manageIndexesApi;
     }
 
-    public IndexModel createIndex(CreateIndexRequest createIndexRequest) throws PineconeValidationException {
+    /**
+     * Creates a new index based on the provided configuration.
+     *
+     * <pre>
+     *     import io.pinecone.clients.Pinecone;
+     *     import org.openapitools.client.model.*;
+     *
+     *     Pinecone client = new Pinecone.Builder("your-api-key").build();
+     *
```

Comment:
Annoying I know, but now we don't need the `serverlessSpec` things. You can just do `client.createServerlessIndex(blahblah)`

Reply:
Yup, this entire thing is getting pulled out once I rebase off your deprecation work.